client/my-sites/plugins/plugin-site-jetpack/index.jsx

```diff
@@ -1,141 +1,130 @@
 import PluginInstallButton from 'calypso/my-sites/plugins/plugin-install-button';
 import PluginRemoveButton from 'calypso/my-sites/plugins/plugin-remove-button';
-import PluginUpdateIndicator from 'calypso/my-sites/plugins/plugin-site-update-indicator';
 import {
 	getPluginOnSite,
 	isPluginActionInProgress,
 } from 'calypso/state/plugins/installed/selectors';
 
 import './style.scss';
 
-class PluginSiteJetpack extends Component {
-	static propTypes = {
-		site: PropTypes.object,
-		plugin: PropTypes.object,
-		allowedActions: PropTypes.shape( {
-			activation: PropTypes.bool,
-			autoupdate: PropTypes.bool,
-			remove: PropTypes.bool,
-		} ),
-		isAutoManaged: PropTypes.bool,
-	};
+const PluginSiteJetpack = ( props ) => {
+	const translate = useTranslate();
+	const installInProgress = useSelector( ( state ) =>
+		isPluginActionInProgress( state, props.site.ID, props.plugin.id, INSTALL_PLUGIN )
+	);
+	const pluginOnSite = useSelector( ( state ) =>
+		getPluginOnSite( state, props.site.ID, props.plugin.slug )
+	);
+	const [ isMobileLayout, setIsMobileLayout ] = useState();
 
-	static defaultProps = {
-		allowedActions: {
-			activation: true,
-			autoupdate: true,
-			remove: true,
-		},
-		isAutoManaged: false,
-	};
-
-	renderInstallButton = () => {
-		return (
-			<PluginInstallButton
-				isEmbed={ true }
-				selectedSite={ this.props.site }
-				plugin={ this.props.plugin }
-				isInstalling={ this.props.installInProgress }
-			/>
-		);
-	};
-
-	renderInstallPlugin = () => {
-		return (
-			<FoldableCard
-				compact
-				className="plugin-site-jetpack"
-				header={ <Site site={ this.props.site } indicator={ false } /> }
-				actionButton={ this.renderInstallButton() }
-			/>
+	useEffect( () => {
+		if ( isWithinBreakpoint( '<1040px' ) ) {
+			setIsMobileLayout( true );
+		}
+		const unsubscribe = subscribeIsWithinBreakpoint( '<1040px', ( isMobile ) =>
+			setIsMobileLayout( isMobile )
 		);
-	};
 
-	renderPluginSite = () => {
-		const {
-			activation: canToggleActivation,
-			autoupdate: canToggleAutoupdate,
-			remove: canToggleRemove,
-		} = this.props.allowedActions;
+		return () => {
+			if ( typeof unsubscribe === 'function' ) {
+				unsubscribe();
+			}
+		};
+	}, [] );
 
-		const showAutoManagedMessage = this.props.isAutoManaged;
-
-		const settingsLink = this.props?.pluginOnSite?.action_links?.Settings ?? null;
+	if ( ! props.site || ! props.plugin ) {
+		return null;
+	}
 
+	if ( ! pluginOnSite ) {
 		return (
-			<FoldableCard
-				compact
-				clickableHeader
-				className="plugin-site-jetpack"
-				header={ <Site site={ this.props.site } indicator={ false } /> }
-				summary={
-					<PluginUpdateIndicator
-						site={ this.props.site }
-						plugin={ this.props.plugin }
-						expanded={ false }
-					/>
-				}
-				expandedSummary={
-					<PluginUpdateIndicator
-						site={ this.props.site }
-						plugin={ this.props.plugin }
-						expanded={ true }
-					/>
-				}
-			>
```

Comment:
We are now missing the `Update` button. I have managed to see this only in a Jetpack site with an older plugin version eg `https://downloads.wordpress.org/plugin/cloudflare.4.5.1.zip`

![SS 2021-11-25 at 11 21 36](https://user-images.githubusercontent.com/12430020/143414408-fdbd0bcd-c2c8-454f-87f2-82cdac1fe1ff.png)

Reply:
> We are now missing the Update button

Do we have layouts about how to display this option on the new grid?

Reply:
no we don't have any designs. I think we can add an "Update to #" button (styled like a text link)
next to the Autoupdate toggle when autoupdate is off?

![SS 2021-11-26 at 15 27 18](https://user-images.githubusercontent.com/12430020/143588060-3399194a-fe20-4878-bbf8-3664aed39db5.png)

Reply:
Applied at 1e2cdd8c52390203443d74aace3439fb11a39105.
![Screen Shot 2021-11-29 at 04 15 07](https://user-images.githubusercontent.com/5039531/143833610-a8ce5beb-1cd5-4c5f-a0f6-fcc44dd5a6b6.png)
![Screen Shot 2021-11-29 at 04 14 57](https://user-images.githubusercontent.com/5039531/143833620-bba0cf8e-3ed2-463f-adaf-9537e23b9808.png)
![Screen Shot 2021-11-29 at 04 14 45](https://user-images.githubusercontent.com/5039531/143833626-dbc35423-b41b-4413-a19d-c228b983e45c.png)